index.html

```diff
@@ -8,340 +8,305 @@
-        <figure class="img_placeholder">
-          <img src="/images/placeholder.png" alt="">
-        </figure>
+  
+      <section class="projects" id="project-s">
+        <article class="recent_work">
+          <h2 class="recent-mobile">MY RECENT WORK</h2>
+          <h2 class="recent-desk">My recent work</h2>
+        </article>
+  
+        <section id="project-list">
+          <!-- <section class="ellipse_19">
+            <figure class="img_placeholder">
+              <img class="imagePlaceholder" src="/images/placeholder.png" alt="">
+            </figure>
     
-        <section class="post_info">
-          <article>
-            <h2 class="title_post">Multi-Post Stories</h2>
-          </article>
+            <section class="post_info">
+              <article>
+                <h2 class="title_post">Multi-Post Stories</h2>
+              </article>
+              
+              <article>
+                <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. has been the 
+                  industry's standard dummy text ever since the 1500s, 
+                  when an unknown printer took a standard dummy text.
+                </p>
+              </article>
+          
+              <article class="tags">
+                <ul>
+                  <li class="tech-item">css</li>
+                  <li class="tech-item">html</li>
+                  <li class="tech-item">javascript</li>
+                  <li class="tech-item">Ruby</li>
+                </ul>
+              </article>
+          
+              <section class="button">See Projects</section>
+            </section>
+          </section> -->
   
-          <div class="ellipse_21"></div>
+          <!-- <section class="ellipse_19">
+            <figure class="img_placeholder">
+              <img class="imagePlaceholder" src="/images/placeholder.png" alt="">
+            </figure>
+    
+            <section class="post_info">
+              <article>
+                <h2 class="title_post">Multi-Post Stories</h2>
+              </article>
+              
+              <article>
+                <p>
+                  A daily selection of privately personalized reads; 
+                  no accounts or sign-ups required. has been the 
+                  industry's standard dummy text ever since the 1500s, 
+                  when an unknown printer took a standard dummy text.
+                </p>
+              </article>
+          
+              <article class="tags">
+                <ul>
+                  <li class="tech-item">css</li>
+                  <li class="tech-item">html</li>
+                  <li class="tech-item">javascript</li>
+                  <li class="tech-item">Ruby</li>
+                </ul>
+              </article>
+          
+              <section class="button">See Projects</section>
+            </section>
+          </section> -->
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻